rootfs/rootfs/etc/rc.d/prltoolsd

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/bin/sh /usr/local/etc/init.d/prltoolsd start
```

Comment:
Is this file not executable as-is?  Why the explicit call out to `/bin/sh`?  Do we need a `-a` on the `cp` line where this comes in?

Also, just to be clear, if I run this daemon on VMware or VirtualBox, will the daemon stay resident, or will it terminate appropriately?